bin/pint

```diff
@@ -63,16 +72,22 @@ options:
 """
 
 import sys
+import json
 
 from docopt import docopt
 
 import susepubliccloudinfoclient.infoserverrequests as ifsrequest
 import susepubliccloudinfoclient.version as version
 
-command_args = docopt(__doc__, version=version.VERSION)
+provider_data = ifsrequest.get_provider_data(None, None, 'json', 'all', None)
+provider_dict = json.loads(provider_data)
```

Comment:
Please do not include the data type in the variable name this is not consistent with the style used anywhere else in the code. `providers` instead of `providers_dict` will work.